code/components/gta-core-five/src/DisableCameraShake.cpp

```diff
@@ -6,40 +6,45 @@
  */
 
 #include <StdInc.h>
-#include <GameInit.h>
 #include <Hooking.h>
 #include <CoreConsole.h>
-#include <nutsnbolts.h>
+
+#include "Hooking.Stubs.h"
 
 static std::shared_ptr<ConVar<bool>> g_cameraShakeConvar;
 
-static void CameraShakeOverride()
+static void (*g_origUpdateCameraVehicleHighSpeedShake)(void*);
+static void UpdateCameraVehicleHighSpeedShake(void* a1)
 {
-	if (g_cameraShakeConvar->GetValue())
+	if (!g_cameraShakeConvar->GetValue())
```

Comment:
Feels like `g_disableCameraShakeConvar` would be a better name to reflect the value of this convar meaning. Since this is a rework - we can improve this as well.

Reply:
Even better: how about not calling GetValue on the convar in the hook but instead using the tracking variable support so that there's only a single `bool` check?